src/main/java/no/fdk/dataservicecatalog/service/ApiHarvesterReactiveClient.java

```diff
@@ -18,19 +18,20 @@
     private final WebClient webClient;
 
     ApiHarvesterReactiveClient(ApiHarvesterProperties properties) {
-        webClient = WebClient.builder().defaultHeader("accept", MediaType.APPLICATION_JSON_VALUE).baseUrl(properties.getUrl()).build();
+        webClient = WebClient.builder().defaultHeader("accept", MediaType.APPLICATION_JSON_VALUE).build();
```

Comment:
Can we build this client where it is needed? This constructor seems unnecessary here.

Reply:
Yes!